flexmeasures/ui/templates/crud/asset.html

```diff
@@ -353,6 +374,99 @@ <h3>All child assets for {{ asset.name }}</h3>
+                          role="button"
+                          data-bs-toggle="dropdown"
+                          aria-expanded="false"
+                          rel="tooltip"
+                          aria-hidden="true"
+                          tabindex="0"
+                          data-bs-placement="right"
+                          data-bs-toggle="tooltip"
+                        ></span>
+                      
+                        <div class="dropdown-menu p-3" style="width: 400px;">
+                            <div>
+                                <strong>Dashboard Graphs Help</strong>
+                                <p class="mb-2">
+                                  Here you can edit what data is shown in the Dashboard Graphs. Each graph can show one or more sensors 
+                                  (<em>it makes sense to show sensors in a graph that share the same unit</em>).
+                                </p>
+                                <p class="mb-2">
+                                  You can also set the title of each graph and re-order them. Select a graph 
+                                  <span class="text-primary">(by clicking on its card)</span> to add sensors to it.
+                                </p>
+                                <p class="mb-0">
+                                  Sensors can be searched on the right. This will list sensors on the asset or its child asset, as well as public assets.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                      
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                  </div>
+                <div class="modal-body">
+                    <div class="row">
+                        <div class="col">
+
+                            <button class="btn btn-primary mb-3" onclick="addNewGraph()">
+                                Add Graph
+                            </button>
+
+                            <div id="graphList" class="row"></div>
+                        </div>
+                      
+                        <div class="col">
+                            <div class="row mb-3">
+                                <div class="col-8">
+                                    <input
+                                        type="text"
+                                        id="searchInput"
+                                        class="form-control"
+                                        placeholder="Search sensors..."
+                                        oninput="filterSensors()"
+                                    />
+                                </div>
+                        
+                                <div class="col-4">
+                                    <select class="form-select" aria-label="Default select example"
```

Comment:
In one of the last commits, you removed  `id="unitsSelect"` from this `select` tag, which stopped the sensor search from working. Adding it back fixed it for me. 
So please add it back :)

Reply:
yes, I added it back as I noticed the search wasn't working